guvnor-structure/guvnor-structure-api/src/main/java/org/guvnor/structure/repositories/PullRequest.java

```diff
@@ -0,0 +1,16 @@
+package org.guvnor.structure.repositories;
+
+public interface PullRequest {
+
+    long getId();
```

Comment:
Hmm and should it also contain the list of commits that you want to include in the PR? or how is that calculated?